src/components/Settings/Display/Display.messages.js

```diff
@@ -95,5 +95,14 @@ export default defineMessages({
     id: 'cboard.components.Settings.Display.darkThemeSecondary',
     defaultMessage:
       'The theme specifies the color of the components, darkness of the surfaces, level of shadow, appropriate opacity of ink elements, etc.'
+  },
+  enableOutputExpansion: {
+    id: 'display.enableOutputExpansion',
+    defaultMessage: 'Enable output expansion'
+  },
+  enableOutputExpansionDescription: {
+    id: 'display.enableOutputExpansionDescription',
+    defaultMessage:
+      'Show a toggle button to expand the output bar for better visibility.'
```

Comment:
I think there is not a clear instruction to define new messages in the project sorry for that.
We need need to add this messages to `src/translations/src/cboard.json` file.